### ISSUE TITLE
New feature for storing ip

### ISSUE THREAD

**Issue by Sarfroz** [skydiver/laravel-route-blocker#19]
Can you add a feature to fetch IP from the database table?

**Comment by ritaorion** [comment]
I agree with Sarfroz. That would be nice. I love the simplicity of this, but it would be really nice if there also was a facade to add blacklisted IP's to the DB. Food for thought. Keep up the good work!
